### PR TITLE
AUDIO: Series page - make contributions banner fancy

### PR DESCRIPTION
**Pull request by ajwl**
## What does this change?
Currently this is just a black box with text. This adds some colour, an svg and a border thing. 

*To preview* - this only visible after 5 episodes have gone up. The best way to check it is on a series with more than 5 episodes. Go to 
[AudioFlagship.scala](https://github.com/guardian/frontend/blob/master/common/app/conf/AudioFlagship.scala) and change the `tagName` to `books/series/books`

## Screenshots

BEFORE 
screen shot 2018-10-30 at 13 51 43

AFTER
screen shot 2018-10-30 at 13 16 48

screen shot 2018-10-30 at 13 17 12

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ ] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [ ] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [ ] Locally
- [ ] On CODE (optional)

### CODE REVIEW

**Review by tomrf1 — APPROVED** [review]
looks good

File: static/src/javascripts/projects/journalism/views/audioSeriesContributions.html [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
-            <h2>Support The Guardian</h2>
-            <p>Help us deliver the independent
-                <br> journalism the world needs</p>
-            <a class="contributions__contribute contributions__option-button" data-link-name="audio-series-contributions">Support the Guardian</a>
+        <div class="fc-container--rolled-up-hide fc-container__body flagship-audio__contributions-banner-holder">
+            <div class="flagship-audio__contributions-banner-text">
+                <h2>Support The Guardian</h2>
+                <p>Help us deliver the independent
+                    <br> journalism the world needs</p>
+                <a class="contributions__contribute contributions__option-button" data-link-name="audio-series-contributions">Support the Guardian</a>
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/journalism/views/audioSeriesContributions.html]: no `href`?

File: static/src/javascripts/projects/journalism/views/audioSeriesContributions.html [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
+            </div>
+            <div class="flagship-audio__contributions-banner-image">
+                <svg xmlns="http://www.w3.org/2000/svg" width="300" height="107"><path d="M0 83.118h12.148v23.824H0zm16.901-36h12.148v59.824H16.901zm16.902-21.177h12.148v81H33.803zm16.901 21.177h12.148v59.824H50.704zM67.606 18h12.148v88.941H67.606zm16.901 41.824h12.148v47.118H84.507zm16.901-12.706h12.148v59.824h-12.148zm16.902-8.471h12.148v68.294H118.31zM135.211 0h12.148v106.941h-12.148zm16.902 49.765h12.148v57.176h-12.148zm16.901-28.059h12.148v85.235h-12.148zm16.901 8.47h12.148v76.765h-12.148zM202.817 0h12.148v106.941h-12.148zm16.901 47.118h12.148v59.824h-12.148zm16.902-21.177h12.148v81H236.62zm16.901-17.47h12.148v98.471h-12.148zm16.902 41.294h12.148v57.176h-12.148zm16.901 33.882h12.148v23.294h-12.148z" fill="#FF6A56"/></svg>
```
Comment (regiskuckaertz) [inline static/src/javascripts/projects/journalism/views/audioSeriesContributions.html]: If it is an SVG we may reuse somewhere else, perhaps move it into `static/src/inline-svgs/journalism/audio`?

File: static/src/stylesheets/module/journalism/_audio-flagship-tag-page-container.scss [hunk 3]
```diff
@@ -95,24 +95,62 @@
...
+                justify-content: center;
+                z-index: 5;
+                svg {
+                    justify-content: center;
+                    position: absolute;
+                    bottom: 0;
+                    height: auto;
+                    width: auto;
+                    @include mq($until: phablet) {
+                        height: 64%;
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/journalism/_audio-flagship-tag-page-container.scss]: Interesting use of percentages.

File: static/src/javascripts/projects/journalism/views/audioSeriesContributions.html (hunk 1 above)
Comment (ajwl) [reply static/src/javascripts/projects/journalism/views/audioSeriesContributions.html]: eek! good catch. Must have removed this at some point

File: static/src/javascripts/projects/journalism/views/audioSeriesContributions.html (hunk 2 above)
Comment (ajwl) [reply static/src/javascripts/projects/journalism/views/audioSeriesContributions.html]: Yes, fair enough. So I could use an image tag for that? ``

File: static/src/stylesheets/module/journalism/_audio-flagship-tag-page-container.scss (hunk 3 above)
Comment (ajwl) [reply static/src/stylesheets/module/journalism/_audio-flagship-tag-page-container.scss]: ahaha. Yeah. they worked. Is it better to use pixels?

Comment (ajwl) [reply static/src/stylesheets/module/journalism/_audio-flagship-tag-page-container.scss]: I'll make them both 60% to be more regular

File: static/src/javascripts/projects/journalism/views/audioSeriesContributions.html (hunk 2 above)
Comment (ajwl) [reply static/src/javascripts/projects/journalism/views/audioSeriesContributions.html]: In fact, I will leave this as a embedded SVG. Connecting this lodash template up with the correct external file gets fiddly.

**Comment by ajwl** [comment]
If everyone is happy with the changes, I will merge

**Pull request merged**